ApplicationLibCode/Application/RiaSummaryDefines.cpp

```diff
@@ -28,6 +28,15 @@ void caf::AppEnum<RiaDefines::HorizontalAxisType>::setUp()
     addItem( RiaDefines::HorizontalAxisType::SUMMARY_VECTOR, "SUMMARY_VECTOR", "Summary Vector" );
     setDefault( RiaDefines::HorizontalAxisType::SUMMARY_VECTOR );
 }
+
+template <>
+void caf::AppEnum<RiaDefines::AccumulatedOrRateType>::setUp()
```

Comment:
AccumulatedOrRateType is a interesting name. CurveType, DataType, AccumulationType?

Reply:
Agree, changed